.gitignore

```diff
@@ -7,6 +7,7 @@ yarn-error.log
 .svn
 composer.lock
 vendor/
+/nbproject
```

Comment:
I recommend that you add the /nbrpoject to your global gitignore, that way you won't have to add it to the project. Or you can move it to where the `.idea` folder is ignored